deploy/200-role.yaml

```diff
@@ -21,7 +21,11 @@ metadata:
 rules:
 - apiGroups: ['shipwright.io']
   resources: ['buildruns']
-  verbs:     ['get', 'list', 'update', 'watch']
+  verbs:     ['get', 'list', 'watch', 'update']
```

Comment:
Why was this order changed?

Reply:
My first job as an engineer involved managing a database, so I always think of permissions in the escalating order of "read", "create", "update", "delete"

Reply:
That's reasonable. I'll see if I can get rbac-audit to sort these verbs with the same semantics.